### ISSUE TITLE
Add repl_rt_cascading_rtq test

### ISSUE THREAD

**Issue by kellymclaughlin** [basho/giddyup#123]
Works with any backend, riak_ee only, any version

https://github.com/basho/riak_test/blob/master/tests/repl_rt_cascading_rtq.erl

**Comment by seancribbs** [comment]
As advised in Hipchat, applying to 2.0 only.

**Comment by seancribbs** [comment]
Deployed, fixed my booboo on #122

**Comment by seancribbs** [comment]
```
Creating test repl_aae_fullsync_custom_n with {"platform"=>"ubuntu-1004-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"ubuntu-1004-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"ubuntu-1204-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"ubuntu-1204-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"fedora-17-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"fedora-17-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"centos-5-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"centos-5-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"centos-6-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"centos-6-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"solaris-10u9-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"solaris-10u9-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"freebsd-9-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"freebsd-9-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_aae_fullsync_custom_n with {"platform"=>"osx-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
Creating test repl_rt_cascading_rtq with {"platform"=>"osx-64", "min_version"=>"2.0.0"} for projects ["riak_ee"]
```
